<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wDevTools</class>
 <widget class="QMainWindow" name="wDevTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wDevTools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources.qrc">
    <normaloff>:/rec/data/icons/iconToolbox.svg</normaloff>:/rec/data/icons/iconToolbox.svg</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gbxFunctions">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string notr="true">Functions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnGetListOfFiles">
           <property name="text">
            <string notr="true">Get list of files</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnSoundFileLister">
           <property name="text">
            <string notr="true">Get sound file paths from sound.cfg</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btnLineCounter">
             <property name="text">
              <string notr="true">Count lines from file(s)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLineCounterStart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnGetKeywords">
           <property name="text">
            <string notr="true">Get all keywords from file(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string notr="true">Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbnOutputToGUI">
           <property name="text">
            <string notr="true">Output to GUI</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbnOutputToFile">
           <property name="text">
            <string notr="true">Output to file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="ledOutputPath">
           <property name="text">
            <string notr="true">C:/OTDevToolsOutput.txt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbnOutputPathSelection">
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="pteOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pgbProgress">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
